Apps/WebClient/src/ClientApp/src/models/laboratory.ts

```diff
@@ -35,3 +35,10 @@ export interface LaboratoryReport {
     encoding: string;
     data: string;
 }
+
+export abstract class LaboratoryUtil {
+    public static isTestResultReady(testStatus: string | null): boolean {
+        if (testStatus == null) return false;
+        else return ["Final", "Corrected", "Amended"].includes(testStatus);
```

Comment:
using only the includes looks cleaner since it will return false if is null as well

Reply:
@FuriousLlama array.includes doesn't seem to like nullable string. Can you confirm?
![image](https://user-images.githubusercontent.com/48332333/109895551-427c6100-7c44-11eb-8d8b-1f4a995cf2e7.png)

Reply:
Thanks for checking! Then lets use brackets on the if, else and we are good to go!

Reply:
Thanks @FuriousLlama . I have added the brackets.